shinhye/db/migrations/20230126075942_create_posts_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE posts (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_id INT NOT NULL,
+    user_profile_image VARCHAR(512) NULL,
+    post_image_url VARCHAR(512) NOT NULL,
+    post_content VARCHAR(512) NULL,
```

Comment:
컬렴명 수정해주세요!

작성하신 부분처럼 컬럼명을 관리한다면 `posts_post_image_url`로 조회하게 됩니다.

`posts_image_url`로 조회했을 때도 그 의미와 결과값을 예측할 수 있기 때문입니다.

해당 부분과 관련된 코드를 모두 수정해주세요!
```suggestion
    image_url VARCHAR(512) NOT NULL,
    content VARCHAR(512) NULL,
```

Reply:
넵 반영했습니다!